ops/README.md

```diff
@@ -0,0 +1,23 @@
+OPS
+===
+
+Description of each folder
+
+## ekl
+EKL abbreviation for Elastic Search, Kibana and Logstash. It contains
+sample debug configuration for testing EKL locally. It uses
+`docker-compose` to run.
+
+## ensure_kibana_configuration
```

Comment:
i don't think we want this.

Reply:
i mean, i don't think a third option should be available. we need a development and production option only